integration_tests/test_gated.py

```diff
@@ -35,10 +35,16 @@ def test_gated_success(self):
 
     def test_gated_no_login_failure(self):
         self.assertEqual(1, self.gated(
-            'ubuntu-core', expected_error='Have you run "snapcraft login'))
+            'ubuntu-core', expected_output='Have you run "snapcraft login'))
```

Comment:
No, that's not a mismatch, scared me for a bit :-)

The error message has double quotes, the expected output has single quotes, and is not the whole message.